app/controllers/admin/talentlms_courses_controller.rb

```diff
@@ -0,0 +1,53 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin
+  class TalentlmsCoursesController < ApplicationController
+    before_action :require_can_manage_config!
+    before_action :set_course, only: [:update, :edit, :destroy]
+
+    def new
+      config = Talentlms::Config.find(params['config']) if params['config']
```

Comment:
```suggestion
      config = Talentlms::Config.find(params['config'].to_i) if params['config']
```

Reply:
or maybe config is a string?  If it is, this should get passed through a method that explicitly specifies acceptable values.

Reply:
Looks like I missed cleaning up this one from when I was taking a different approach that had the courses being built from within the config. This line is unnecessary now - I'll strip it out in the next commit.